build.sbt

```diff
@@ -483,6 +483,10 @@ lazy val docs = project("docs")
         writer = new Writer(serializerPlugins = Writer.defaultPlugins(paradoxDirectives.value)))
     },
     paradoxGroups := Map("Language" -> Seq("Scala", "Java")),
+    paradoxRoots := List(
+      "index.html",
+      "quickstart-java/index.html",
+      "quickstart-scala/index.html"),
```

Comment:
What does this lead to, that we don't list the quickstarts in the regular navigation tree but have separate navs for those?

Reply:
It's pretty good to not have them included in ToC because then they become "standalone" without distraction of other things.

Reply:
Was more looking for confirmation that separate ToC trees is what it means, wasn't sure.